### PR TITLE
Add configmap for grafana dashboard

### PR DESCRIPTION
**Pull request by tei-k**
# Description
https://scalar-labs.atlassian.net/browse/DLT-9612

ref: https://gist.github.com/superbrothers/25581ba4204bad45b6adef1855678010

### CODE REVIEW

File: charts/scalardl/values.yaml [hunk 1]
```diff
@@ -83,6 +83,12 @@ envoy:
     # envoy.prometheusRule.namespace -- which namespace prometheus is located. by default monitoring
     namespace: monitoring
 
+  grafanaDashboard:
+    # ledger.grafana_dashboard.enabled -- enable grafana dashboard
+    enabled: false
+    # ledger.grafana_dashboard.namespace -- which namespace grafana dashboard is located. by default monitoring
```
Comment (superbrothers) [inline charts/scalardl/values.yaml:89]:
```suggestion
    # ledger.grafanaDashboard.enabled -- enable grafana dashboard
    enabled: false
    # ledger.grafanaDashboard.namespace -- which namespace grafana dashboard is located. by default monitoring
```

File: charts/scalardl/values.yaml [hunk 2]
```diff
@@ -186,6 +192,12 @@ ledger:
     # ledger.prometheusRule.namespace -- which namespace prometheus is located. by default monitoring
     namespace: monitoring
 
+  grafanaDashboard:
+    # ledger.grafana_dashboard.enabled -- enable grafana dashboard
+    enabled: false
+    # ledger.grafana_dashboard.namespace -- which namespace grafana dashboard is located. by default monitoring
+    namespace: monitoring
```
Comment (superbrothers) [inline charts/scalardl/values.yaml:199]:
```suggestion
    # ledger.grafanaDashboard.enabled -- enable grafana dashboard
    enabled: false
    # ledger.grafanaDashboard.namespace -- which namespace grafana dashboard is located. by default monitoring
    namespace: monitoring
```

**Review by superbrothers — CHANGES_REQUESTED** [review]

**Review by superbrothers — APPROVED** [review]

**Review by ymorimo — APPROVED** [review]
LGTM!

File: charts/scalardl/files/scalardl_grafana_dashboard.json [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+      "6h",
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "",
+  "title": "Scalar DLT Response",
```
Comment (feeblefakie) [inline charts/scalardl/files/scalardl_grafana_dashboard.json:244]:
```suggestion
  "title": "Scalar DL Response",
```

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
Left a minor naming suggestion.

File: charts/scalardl/files/scalardl_grafana_dashboard.json (hunk 3 above)
Comment (feeblefakie) [reply charts/scalardl/files/scalardl_grafana_dashboard.json]: @tei-k Applied it by myself.

**Pull request merged**